.circleci/config.yml

```diff
@@ -250,14 +250,43 @@ jobs:
     executor: docker-typescript-builder
     steps:
       - checkout
+      - aws-setup
+      - aws-ecr-setup
+      - local-deploy-setup
+      - run: echo "export IMAGE_TAG=dev_$(git rev-parse --short=8 HEAD)" >> $BASH_ENV
+      # since we're running with `--build-all`, assume that if it passes, we have all images required for Forge
+      - run: aws ecr describe-images --repository-name="aptos/validator" --image-ids=imageTag=$IMAGE_TAG
+      - run:
+          name: Start local testnet docker-compose
+          shell: /bin/bash
+          working_directory: docker/compose/validator-testnet
+          command: |
+            export VALIDATOR_IMAGE_REPO=${AWS_ECR_ACCOUNT_URL}/aptos/validator
+            export FAUCET_IMAGE_REPO=${AWS_ECR_ACCOUNT_URL}/aptos/faucet
+            docker-compose up -d
+      - run:
+          name: Verify testnet docker-compose up
```

Comment:
To avoid copy-pasta, either move this to a separate modular command that can be reused, or test it in the same job as the docker-compose